### DISCUSSION TITLE
ON/OFF State not changing

### DISCUSSION THREAD

**Discussion by CrownCityMisfit** [nguyenquyhy/Flight-Tracker-StreamDeck#170] [Q&A] [answered]
First of all, thank you for this amazing plugin.

I am setting up my first button - taxi lights. I dropped in the appropriate event and it works! The only problem I have is that my custom graphics are not switching ON and OFF states. The button displays the OFF state only, even after clicking it, and seeing it change to ON in the sim.

I tried embedding and linking the images without success.

The only field I edited is Toggle Event - TOGGLE_TAXI_LIGHTS

Do I need to do anything else?

Cheers.
![172_taxi_OFF](https://user-images.githubusercontent.com/32145070/106652708-b561c180-654a-11eb-8446-0c6912a29d85.jpg)

![172_taxi_ON](https://user-images.githubusercontent.com/32145070/106652723-bb57a280-654a-11eb-892e-0c183b589cf5.jpg)

**Comment by nguyenquyhy** [comment, marked as answer]
If you want to use ON/OFF images, you have to set "Feedback value" to appropriate SimConnect variable (e.g. `LIGHT TAXI` or `LIGHT TAXI ON`. You can read more details on "Feedback value" in our [User Guide](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/master/docs/USERGUIDE.md).

**Reply by CrownCityMisfit** [reply]
Thanks so much!
